sdk/typescript/src/builder/TransactionBlock.ts

```diff
@@ -105,8 +105,8 @@ const MAX_GAS_OBJECTS = 256;
 // The maximum gas that is allowed.
 const MAX_GAS = 1000000000;
 
-// A guess about how much overhead each coin provides for gas calculations.
-const GAS_OVERHEAD_PER_COIN = 10n;
+// An amount of gas (in gas units) that is added to transactions as an overhead to ensure transactions do not fail.
+const GAS_SAFE_OVERHEAD = 10n;
```

Comment:
changing this to 1000 could be a good first cut. More conservative and still very close to reality...

Reply:
Updated to 1000